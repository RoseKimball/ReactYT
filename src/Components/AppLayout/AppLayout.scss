@import '../../styles/Shared.scss';

.app-layout {
    & > *:not(:first-child) {
        margin-top: $header-height;
    }

    ::-webkit-scrollbar {
        width: 6px;
    }
}