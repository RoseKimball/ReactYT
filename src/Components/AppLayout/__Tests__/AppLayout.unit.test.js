import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import AppLayout from '../AppLayout';

describe('AppLayout', () => {
    test('renders layout without children', () => {
        const wrapper = shallow (
            <AppLayout />
        )
        expect(wrapper).toMatchSnapshot();
    })

    test('renders layout with one child', () => {
        const wrapper = shallow (
            <AppLayout>
                <div>Child</div>
            </AppLayout>
        );
        expect(wrapper).toMatchSnapshot();
    })

    test('renders layout with multiple children', () => {
        const wrapper = shallow (
            <AppLayout>
                <div>
                    1st child
                    <div>2nd child</div>
                </div>
            </AppLayout>
        )
    })
})