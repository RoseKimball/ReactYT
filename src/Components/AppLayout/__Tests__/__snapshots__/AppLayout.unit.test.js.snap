// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout renders layout with one child 1`] = `ShallowWrapper {}`;

exports[`AppLayout renders layout without children 1`] = `ShallowWrapper {}`;
